<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PozycjaDialog</class>
 <widget class="QDialog" name="PozycjaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="ProductLabel">
     <property name="text">
      <string>Product:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="ProductComboBox"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="CountLabel">
     <property name="text">
      <string>Count:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="CountBox">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::UpDownArrows</enum>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="TotalCostLabel">
     <property name="text">
      <string>Total cost:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="TotalCostEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PozycjaDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PozycjaDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ProductComboBox</sender>
   <signal>activated(int)</signal>
   <receiver>PozycjaDialog</receiver>
   <slot>onIndexChange(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CountBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>PozycjaDialog</receiver>
   <slot>onCountChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>onIndexChange(int)</slot>
  <slot>onCountChanged(int)</slot>
 </slots>
</ui>
